src/Internal.AspNetCore.BuildTools.Tasks/build/Tasks.tasks

```diff
@@ -11,4 +11,7 @@
   <UsingTask TaskName="Microsoft.AspNetCore.BuildTools.SetEnvironmentVariable" AssemblyFile="$(_BuildToolsAssembly)" />
   <UsingTask TaskName="Microsoft.AspNetCore.BuildTools.WaitForDebugger" AssemblyFile="$(_BuildToolsAssembly)" />
   
+  <!-- TODO remove when XmlPoke becomes available in MSBuild 15.2 -->
```

Comment:
Can we file a work item for this (if it isn't already there) and remove the TODO?